browser.js

```diff
@@ -307,9 +307,11 @@ function noop () {}
 /* eslint-disable */
 /* istanbul ignore next */
 function pfGlobalThisOrFallback () {
-  const defd = (o) => typeof o !== 'undefined' && o
+  var defd = function (o) {
+    return typeof o !== 'undefined' && o;
+  }
   try { 
-    if (typeof globalThis !== 'undefined') return globalThis
+    if (typeof globalThis !== 'undefined') return globalThis;
```

Comment:
stray ;

Reply:
Whitespace languages are a pet peeve, but let me change it :)

Reply:
Done.